### PR TITLE
Initialise

### PR DESCRIPTION
**Pull request by mattgarner**
This change is [Reviewable](https://reviewable.io/reviews/eastgenomics/dnanexus_nirvana/2)

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<!-- dx-header -->
+This is the source code for an app that runs on the DNAnexus Platform.
+For more information about how to run or modify it, see
+https://documentation.dnanexus.com/.
+<!-- /dx-header -->
+
+# dnanexus_nirvana_2.0.3 (https://github.com/Illumina/Nirvana/releases/tag/v2.0.3)
```
Comment (woook) [inline Readme.md:7]: This is ~18 months behind the most recent release. Could we raise an issue for this repo to update Nirvana? I realise that we may not be sticking with Nirvana for long but feel the issue should be logged.

File: Readme.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+## What does this app output?
+The app outputs one file, where [outPrefix] is the vcf filename without extension:
+1. [outPrefix].json.gz - Annotation for variants present within the input vcf (gzipped json file)
+
+## How does this app work?
+The app runs Nirvana 2.0.3 using an input VCF file and uploads the output to DNAnexus.
+
+## What are the limitations of this app
+- Only features present in the annotation data will be annotated against variants. i.e. if a transcript is not present within the annotation data then it will be absent from the annotation of variants within that transcripts
+- This version (2.0.3) of Nirvana is outdated, and contains known bugs including incorrect HGVS annotation of variants downstream of differences between the reference genome and the transcript sequence.
```
Comment (woook) [inline Readme.md:35]: See comment above!

File: dxapp.json [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  },
+    "access": {
+    "project": "CONTRIBUTE",
+    "allProjects": "VIEW",
+    "network": [
+      "*"
+    ]
+  },
+  "regionalOptions": {
+    "aws:us-east-1": {
```
Comment (woook) [inline dxapp.json:48]: Will still app run in eu-central-1 (frankfurt)?
Can we add that as an option now?
I think some of the reference files are only in us-east-1 ...

File: dxapp.json [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+      "systemRequirements": {
+        "*": {
+          "instanceType": "mem1_ssd1_v2_x4"
```
Comment (woook) [inline dxapp.json:51]: Did you try other configurations?

**Review by woook — COMMENTED** [review]
I've made some comments but looks almost ready to go

File: Readme.md (hunk 1 above)
Comment (mattgarner) [reply Readme.md:7]: Issue created #3

File: Readme.md (hunk 2 above)
Comment (mattgarner) [reply Readme.md:35]: Issue created #3

File: dxapp.json (hunk 3 above)
Comment (mattgarner) [reply dxapp.json:48]: Correct, refs are in US right now. We don't have a dedicated refs project set up in Frankfurt yet, so I haven't updated the links to these files to use eu. Could store them in the existing Frankfurt project temporarily for now, but would still need to update the project:file again later. Similarly I have not changed the region to eu as the ref data is stuck in US. They're too big to store with the app (~25G).

File: dxapp.json (hunk 4 above)
Comment (mattgarner) [reply dxapp.json:51]: This is the smallest workable instance, fewer cores gives insufficient disk space. Most of the run time is taken up by downloading and unpacking the data (~13 of 15 mins). Once everything is set up the annotation only takes ~2 mins for a clinical exome so I don't expect that we would see much benefit from going higher spec.

Comment (woook) [reply dxapp.json:51]: :+1:

File: dxapp.json (hunk 3 above)
Comment (woook) [reply dxapp.json:48]: I'm thinking that if we need to use this app for patient data, it will have to run in frankfurt...

File: dxapp.json (hunk 4 above)
Comment (woook) [reply dxapp.json:51]: I forgot you can't see jobs costs yet

**Review by woook — CHANGES_REQUESTED** [review]
> Reviewed 5 of 5 files at r1.
> *[Reviewable](https://reviewable.io/reviews/eastgenomics/dnanexus_nirvana/2)* status: all files reviewed, 1 unresolved discussion (waiting on @mattgarner)

___
*[dxapp.json, line 48 at r1](https://reviewable.io/reviews/eastgenomics/dnanexus_nirvana/2#-M-RZfNnIWL-9Qa_lyhk-r1-48:-M-RZuL3DdHkF7gFAXJS:b8q9xzl) ([raw file](https://github.com/eastgenomics/dnanexus_nirvana/blob/b2300120242025a884617bd896cc9474dbd7b9f5/dxapp.json#L48)):*
<details><summary><i>Previously, woook wrote…</i></summary><blockquote>

I'm thinking that if we need to use this app for patient data, it will have to run in frankfurt...
</blockquote></details>

Let's leave this as it and with a bit of luck Frankfurt will be fully functional before we need to use it for real

**Review by woook — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/eastgenomics/dnanexus_nirvana/2)* status: :shipit: complete! all files reviewed, all discussions resolved

**Pull request merged**